http/base/rpc/rpc.go

```diff
@@ -98,6 +98,10 @@ func Handle(w http.ResponseWriter, r *http.Request) {
 		log.Error("HTTP JSON RPC Handle - ioutil.ReadAll: ", err)
 		return
 	}
+	if len(body) > 1024*1024 {
```

Comment:
Defined one const variable for it, with (1 << 20).
Or you can use http.DefaultMaxHeaderBytes